File: code_wars/kyu_7/014-Reverse_a_Number.js
// ###### Solution 1
function reverseNumber(n) {
  let result = "";
  const sign = n < 0 ? -1 : 1;
  n = String(Math.abs(n));

  for (let i = 0; i < n.length; i++) result = n[i] + result;

  return sign * Number(result);
}
// ###### Video Link:
// ###### Problem Link: https://www.codewars.com/kata/555bfd6f9f9f52680f0000c5/train/javascript

File: code_wars/kyu_7/014-Reverse_a_Number.py
###### Solution 1
def reverse_number(n: int) -> int:
    result = ""
    sign = -1 if n < 0 else 1

    for digit in str(abs(n)):
        result = digit + result

    return sign * int(result)


###### Solution 2
def reverse_number(n: int) -> int:
    return int((n < 0) * "-" + str(abs(n))[::-1])


###### Video Link:
###### Problem Link: https://www.codewars.com/kata/555bfd6f9f9f52680f0000c5/train/python
